birdhouse/components/finch/default.env

```diff
@@ -6,6 +6,7 @@
 
 export FINCH_VERSION=0.9.2
 export FINCH_IMAGE="birdhouse/finch:version-${FINCH_VERSION}"
```

Comment:
I just realized that all xxx_IMAGE and xxx_URI vars should both be delayed eval because they use xxx_VERSION var and that can be overridden by `env.local`.